app/src/androidTest/java/ch/epfl/sdp/musiconnect/VideoPlayingTests.java

```diff
@@ -90,7 +91,9 @@ public static void goToMyProfilePage() {
         appCompatTextView.perform(click());
     }
 
-    private String videoSource = "https://sites.google.com/site/androidexample9/download/RunningClock.mp4";
+    private String packageName = "ch.epfl.sdp.musiconnect";
+    private String videoSource = "android.resource://"+packageName+"/"+ R.raw.minion;
```

Comment:
You can use a codding method to avoid use of hard coded strin,  like this :
https://stackoverflow.com/questions/3705711/getting-package-string-from-android-manifest

Reply:
the tests and actual code are not in the same package. But I can try to find a way to do it